src/System.Runtime.InteropServices.PInvoke/tests/System.Runtime.InteropServices.PInvoke.Tests.builds

```diff
@@ -2,7 +2,9 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <Project Include="System.Runtime.InteropServices.PInvoke.Tests.csproj"/>
+    <Project Include="System.Runtime.InteropServices.PInvoke.Tests.csproj">
+      <OSGroup>Windows_NT</OSGroup>
```

Comment:
I looked at the history and it looks like I disabled this back in Jan because the test project was not getting built for other platforms.  I'll try removing this and see what happens.